ios/iNaturalistReactNative.xcodeproj/project.pbxproj

```diff
@@ -562,7 +562,7 @@
 				COPY_PHASE_STRIP = NO;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
 				ENABLE_TESTABILITY = YES;
-				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "arm64 ";
+				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "";
```

Comment:
any idea why this changed?

Reply:
Since `arm64` being excluded means it wouldn't work on my laptop (M1 Macbook)